src/main/java/betterquesting/client/gui2/GuiQuestLines.java

```diff
@@ -751,21 +753,23 @@ private void refreshClaimAll()
     }
 
     private void openSearch(PanelButton panelButton) {
-        GuiQuestSearch guiQuestSearch = new GuiQuestSearch(this);
-        guiQuestSearch.setCallback(entry -> {
-            openQuestLine(entry.getQuestLineEntry());
-            UUID selectedQuestId = entry.getQuest().getKey();
-            Optional<PanelButtonQuest> targetQuestButton = cvQuest.getQuestButtons().stream().filter(panelButtonQuest -> panelButtonQuest.getStoredValue().getKey().equals(selectedQuestId)).findFirst();
-            targetQuestButton.ifPresent(panelButtonQuest -> {
-                GuiTextureColored newTexture = new GuiTextureColored(panelButtonQuest.txFrame,
-                        new GuiColorPulse(
-                                new GuiColorStatic(255, 220, 115, 255),
-                                new GuiColorStatic(255, 191, 0, 255),
-                                1, 0
-                        ));
-                panelButtonQuest.setTextures(newTexture, newTexture, newTexture);
+        if (searchGui == null) {
```

Comment:
Hmm, would it make sense to move this initialization into `initPanel()`?

Reply:
Good, suggestion. I'll try that.